### ISSUE TITLE
HTML source has a bug with low image per page count

### ISSUE THREAD

**Issue by MrAndre96** [Bionus/imgbrd-grabber#1379]
**Bug description**

When you set HTML as your first source and with the post filtering your image per page count becomes lower than the normal (most of the time 20) it will not download all the images.

**Steps to reproduce**

For example use sankaku complex or anime pictures as source and make sure that the one you use has html as first source
After that search chrono_clock rating:questionable for sankaku or chrono_clock  light_erotic for anime pictures
Now click on get all
Now change by downloads the images per page count to 3 (Which changes normally when you use post filters)
Now you see that it only downloads 3 images instead of the number of images it said (for sankaku 18 or anime pictures 17)

**Expected behavior**

Download all the images instead of just 3

**System information**

Windows 10 with the latest nightly version

**Comment by Bionus** [comment]
The steps you describe are expected behavior. Most HTML sources have a fixed number of images per page, and shouldn't be changed. The bug here is this part: "with the post filtering your image per page count becomes lower than the normal".

**Comment by MrAndre96** [comment]
@Bionus I think that this fix caused a small issue for sankaku for when there is only 1 page.
When you search chrono_clock rating:questionable you will see that there are 18 results but when you click on get all there is a image limit/images per page of 22 instead of 18.
This doesn't happen with other sources like konachan so probably something with Regex.

**Comment by Bionus** [comment]
Indeed, now it counts post-filtered images, but also skipped images from the "Ignore (page 1)" setting:

![image](https://user-images.githubusercontent.com/882719/44101140-0b02310a-9fe7-11e8-9a9a-642ecdd1dead.png)

I'll push a fix :+1:
